### ISSUE TITLE
Add bookmarks functionality to extension

### ISSUE THREAD

**Issue by virejdasani** [virejdasani/LuminousNewTab#6]
I am not familiar with the browser local storage setting and need help to make an interface where users can add, edit and delete bookmarks on the index page of the extension. Any help like YouTube videos, documentation, and contributions are appreciated. Thanks in advance!

**Comment by virejdasani** [comment]
@devanshuyadav, yes please! I saw Helpage and it looks beautiful. I'll go ahead and assign this issue to you. You can go ahead and make a PR. Thanks a lot!

**Comment by virejdasani** [comment]
No rush, take your time :)

**Comment by devanshuyadav** [comment]
@virejdasani, it's not using local storage anymore.

**Comment by devanshuyadav** [comment]
@virejdasani the chrome.storage.local doesn't seem to work. please clarify.

**Comment by virejdasani** [comment]
Hmm, I'm not sure why. Try pulling the latest code from Github. On line 41 in `src/index.js`, it has been used and it works.

**Comment by devanshuyadav** [comment]
![image](https://user-images.githubusercontent.com/76938941/137612943-b17c4f6a-2978-4b35-85d4-d6d2a34af434.png)
It looks different from the actual page, the settings button does not work too.

**Comment by virejdasani** [comment]
@devanshuyadav this is because you are running the extension like a website. You have to install the extension in your browser like an extension:
https://webkul.com/blog/how-to-install-the-unpacked-extension-in-chrome/

**Comment by virejdasani** [comment]
@devanshuyadav this is because you are running the extension like a website. You have to install the extension in your browser like an extension:
Check it out here: https://vimeo.com/634236988

**Comment by virejdasani** [comment]
Yep, sounds good. Also, don't pull the code from the master branch. Pull it from the noBookmarks branch because I tried to implement bookmarks in the master and it didn't go as I had planned. Anyway, make a PR and I'll merge it :)

**Comment by devanshuyadav** [comment]
I did pull from master already.

**Comment by devanshuyadav** [comment]
I can implement bookmarks at any point. I recently tried that onto my [project ](https://devanshuyadav.github.io/helpage/) and it works. Just give me a specific branch where I'll create a PR to..

**Comment by virejdasani** [comment]
If it's possible, can you implement it into the master branch itself? You'll have to delete some code I wrote for the bookmarks though.

**Comment by devanshuyadav** [comment]
Okay, I'll try.

**Comment by virejdasani** [comment]
https://github.com/virejdasani/LuminousNewTab/pull/27
